elasticdl/python/common/hash_utils.py

```diff
@@ -4,3 +4,24 @@
 def string_to_id(name, num):
     h = hashlib.sha256(name.encode("utf-8"))
     return int(h.hexdigest(), base=32) % num
+
+
+def int_to_id(index, num):
+    return index % num
+
+
+def scatter_embedding_vector(values, indices, num):
```

Comment:
It is a util function, better to add a docstring.

Reply:
Done